packages/@glimmer/integration-tests/lib/suites/each.ts

```diff
@@ -165,6 +166,38 @@ export class EachSuite extends RenderTest {
   'it renders all items with duplicate key values'() {
```

Comment:
This test was not testing what it says it was, so I updated it to test _that_ properly (which it does) and added a couple more tests